src/main/java/io/keyko/nevermined/api/config/NeverminedConfig.java

```diff
@@ -272,11 +315,11 @@ public NeverminedConfig setDidRegistryAddress(String address) {
     }
 
     public String getEscrowPaymentConditionConditionsAddress() {
-        return escrowRewardAddress;
+        return escrowPaymentAddress;
     }
 
     public NeverminedConfig setEscrowPaymentConditionConditionsAddress(String address) {
-        this.escrowRewardAddress = address;
+        this.escrowPaymentAddress = address;
```

Comment:
Is the double `Condition` in the name a typo?

Reply:
yes, it is, fixing it